ios/Podfile

```diff
@@ -42,6 +45,28 @@ target 'ReactNativeBase' do
   use_native_modules!
 end
 
+target 'ReactNativeBase-Staging' do
+    # Pods for ReactNativeBase
+    base_pods
+
+    pod 'react-native-config', :path => '../node_modules/react-native-config'
```

Comment:
Why these pods are not in base pods if we are repating them?

Reply:
I just moved all react-native base dependencies to `base_pods` wasn't sure about these three since they are external dependencies. But yeah, we can move them. WDYT?

Reply:
Sure; I am thinking a case where we could differentiate pods by environment, shouldn't be  a common thing I think, but having the definition in the base could help if happens.

Reply:
Yeah, the only scenario that i could think of is having some native logging tool (like crashlytics) that you only want to setup for the prod target 🤔